js/apps/system/_admin/aardvark/APP/frontend/js/views/graphViewer.js

```diff
@@ -1162,6 +1162,20 @@
           buttons,
           tableContent
         );
+        var container = document.getElementById('jsoneditor');
+        this.resize();
+        var options = {
+          onChange: function () {
+          },
+          onModeChange: function (newMode) {
+            void (newMode);
+          },
+          search: true,
+          mode: 'code',
+          modes: ['tree', 'code'],
+          ace: window.ace
+        };
+        this.editor = new JSONEditor(container, options);
```

Comment:
Better option would be to do the initialisation in `modalView.js` instead of here, as it is the same as here:
https://github.com/arangodb/arangodb/pull/11520/files#diff-3838faf03c2751740f0ae29704cd24d4R718

Anyway, maybe then just add a comment that we're fixing/optimising that in the future. To it does not get lost completely.